readme.md

```diff
@@ -394,16 +394,23 @@ entities:
     name: living temperature in Farenheit # Overrides the entity name
     unit_of_measurement: °F # Overrides the unit
     show_value: true # shows the last value as text
+    customdata: |
+      $fn ({states}) => 
+        states.map( ({...attributes}) => ({ extra_attr: "hello" , ...attributes}) )
```

Comment:
Could only add the attribute that is used in the texttemplate? That will make it clearer and more efficient

Reply:
Updated